index.rst

```diff
@@ -11,14 +11,78 @@ Bharat-Sim's documentation!
    :caption: Contents:
 
 
-
-Indices and tables
-==================
-
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
 
-How does this lool?
+
+
+
+SIR Model with Agents
+=====================
+
+The Bharat-sim framework allows you to build agent-based models. Agents, are essentially the main component of the simulation. They are entities which are defined with a bunch of parameters and certain rules which they have to obey as the simulation runs.
+
+In the purview of the SIR Model, the agents are people, which go about their daily lives and follow regular schedules. Here, the parameters defining an agent are a unique agent id, age, their infection status, the time for which they remain infected and so on. The simulation allows you to introduce new parameters as and when required.
+
+The rules governing the behaviour of all agents include the particular schedules which they follow and, the functions which allow the agents to transit from one compartment to the other.
+
+Schedules
+---------
+
+The current schedules of all the agents are stored in the **create12HourSchedules** function. The default function has three schedules:-
+
+1. **employeeSchedule** - This schedule is followed by all agents above the age of 25.
+2. **studentSchedule** - This is followed by all the agents below the age of 25.
+3. **quarantinedSchedule** - This is an intervention which 'tells' all the infected agents to stay home until they recover.
+
+The *myDay* and *myTick* values defined at the start of the code are used to define all schedules. For example, if *myDay* has 2 ticks (each tick is 12 hours long), the **employeeSchedule** says that all employees remain at home during tick 0, and then they spend tick 1 in the office. The infected agents following the **quarantinedSchedule** spend both ticks at home.
+
+Thus, as more and more classes of agents are introduced, they can be made to follow different schedules. These schedules can also be made more complex by changing the number of ticks in a day.
+
+Behaviours
```

Comment:
This section suffers from the same problem I spoke about above: you're simply _describing_ the code file, you aren't quite explaining how one would write it from scratch. This is a particularly important exercise for @SoumilK1 and @prathithbhargav, since the both of you will be using the framework regularly over this semester, and you should be able to explain why every line is used.